src/reducers/Tasks-test.js

```diff
@@ -1,21 +0,0 @@
-const assert = require('assert')
```

Comment:
Re: the commit message

![dog-no-idea](https://cloud.githubusercontent.com/assets/82317/17075049/d116895e-504d-11e6-9219-88425421346c.jpg)